lib/DllReferencePlugin.js

```diff
@@ -113,14 +113,11 @@ class DllReferencePlugin {
 			new ExternalModuleFactoryPlugin(sourceType || "var", externals).apply(
 				normalModuleFactory
 			);
-			const [contentKey] = Object.keys(content);
-			const { id: contentId } = content[contentKey];
```

Comment:
https://github.com/webpack/webpack/pull/10207#discussion_r400033857

@sokra you were right already removed.